dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceInputStream.java

```diff
@@ -2529,8 +2547,13 @@ private String makeTokenRequest()
         return sb.toString();
     }
 
-    private String makeCloseHandleRequest()
+    private String makeCloseHandleRequest(Span span)
     {
+        String traceContextHeader = null;
+        if (span != null) {
+            traceContextHeader = org.hpccsystems.ws.client.utils.Utils.getTraceParentHeader(span);
```

Comment:
I believe you noticed an issue where traceparentheader flags were inconsistent , is that still an issue we need to address?